e2e/scenario/init.go

```diff
@@ -54,4 +54,10 @@ func scenarios() []*Scenario {
 		ubuntu2204CustomCATrust(),
 		ubuntu2204ArtifactStreaming(),
 	}
+
+	for vmSeries, _ := range DefaultGPUSeriesSKU {
+		scenarios = append(scenarios, ubuntu2204gpu(vmSeries))
```

Comment:
why not have a separate function, maybe called `gpuScenarios()`, which also returns a `[]*Scenario` - then we just concat the output of that with this slice using ellipsis operator

Reply:
we can do that concatenation in the InitiScenarioTable func, and keep this func the same I think

Reply:
something like `scenarios := scenarios() ... scenarios = append(scenarios, gpuScenarios()...)`